src/aiscalator/jupyter/command.py

```diff
@@ -66,9 +66,14 @@ def _prepare_docker_env(conf: AiscalatorConfig, program, reason):
         "-p", "10000:8888",
         "-p", "4040:4040"
     ]
+    
     for env in conf.user_env_file(conf.step_field("task.env")):
         if os.path.isfile(env):
             commands += ["--env-file", env]
+    # allow to pass a list of extra options like ["--network", "bridge"]
+    if conf.step_field("task.docker_extra_options"):
```

Comment:
Since this is related to the docker image, I would rather put it in the `_prepare_docker_image_env` function and in the section of `docker_image.docker_extra_options` rather than `task`.

In addition, Instead of `if conf.step_field("task.docker_extra_options")`, it should probably be
`if conf.has_step_field("docker_image.docker_extra_options")` to make this configuration option optional

Reply:
Right, done !